website/ru.json

```diff
@@ -1,35 +1,37 @@
 {
   "global": {
-    "home": "Зашифровать файлы",
-    "about": "О нас",
-    "adventure": "Цели и достижения",
+    "home": "Шифрование файлов",
+    "about": "О Muonium",
+    "adventure": "Хронология",
     "security": "Безопасность",
     "blog": "Блог",
     "donate": "Пожертвовать",
     "help": "Помощь",
     "login": "Войти",
     "register": "Зарегистрироваться",
     "close": "Закрыть",
-    "vote": "Vote"
+    "vote": "Голосовать"
   },
+  
   "home": {
-    "secureStorageSolution": "Безопасное решение для хранения данных",
-    "securedConnections": "Защищенные сети",
-    "securedConnectionsTxt": "Все ваши сети с Muonium защищены TLS 1.2 technology",
-    "endToEndEncryption": "End-to-end шифрование",
-    "endToEndEncryptionTxt": "Все ваши файлы зашифрованы в вашем браузере,поэтому только вы можете их видеть",
+    "secureStorageSolution": "БЕЗОПАСНОЕ РЕШЕНИЕ ДЛЯ ХРАНЕНИЯ ДАННЫХ",
+    "features": "ОСОБЕННОСТИ",
+    "securedConnections": "Безопасные подключения",
+    "securedConnectionsTxt": "Все ваши подключения к Muonium защищены технологией TLS 1.2",
+    "endToEndEncryption": "Сквозное шифрование",
+    "endToEndEncryptionTxt": "Все файлы шифруются в вашем браузере, поэтому только вы можете их видеть",
     "protection": "Защита",
-    "protectionTxt": "Ваши файлы защищены от шпионажа.Только ваш пароль разрешает вам увидеть файлы.",
-    "free": "Бесплатный",
-    "freeTxt": "Интернет приватность это фундаментальное право.Ваши пожертвования финансируют сервера, нашу команду и конфереции(встречи)",
-    "openSource": "OpenSource",
-    "openSourceTxt": "Muonium использует библиотеки open source шифрования, протестированные сообществом свободного программного обеспечения.",
+    "protectionTxt": "Ваши файлы защищены от шпионажа, и только ваш пароль позволяет просматривать файлы",
+    "free": "Бесплатно",
+    "freeTxt": "Конфиденциальность в интернете является одним из основных прав. Ваши пожертвования помогают поддерживать сервера, нашу команду и конфереции (встречи)",
+    "openSource": "Открытый исходный код",
+    "openSourceTxt": "Исходный код Muonium полностью открыт. Вы можете просмотреть наш код на <a href=\"https://github.com/muonium\">GitHub</a>",
     "anonymity": "Анонимность",
-    "anonymityTxt": "Muonium не требует и не собирает личную информацию,вы остаетесь анонимными",
-    "modernity": "Современность",
```

Comment:
We want to say "Modernity", not "Modern", hence, your translations seem false.

Reply:
Frankly in some languages "Modernity" sounds pretty stupid ;)

Reply:
How come?

Reply:
Sometimes it's just - well, it's hard to explain, but you just read it and think "aargh how can you write this"... It depends on which header "style" you are used to, I think. But again, I don't know how it is in Russian

Reply:
Ah yea yea, I think I've got what you mean x)

Reply:
@0ndrey could you tell us why you changed this string please?